### PR TITLE
Update blueprints doc with permission model

### PR DESCRIPTION
**Pull request by peterdeme**
# Description of the change

## Checklist

Please make sure that the proposed change checks all the boxes below before requesting a review:

- [x] I have reviewed the [guidelines for contributing](https://github.com/spacelift-io/user-documentation/blob/main/CONTRIBUTING.md) to this repository.
- [x] The preview looks fine.
- [x] The tests pass.
- [x] The commit history is clean and meaningful.
- [x] The pull request is opened against the `main` branch.
- [x] The pull request is no longer marked as a draft.
- [x] You agree to license your contribution under the [MIT license](https://github.com/spacelift-io/user-documentation/blob/main/LICENSE) to Spacelift (not required for Spacelift employees).

If the proposed change is ready to be merged, please request a review from `@spacelift-io/solutions-engineering`. Someone will review and merge the pull request.

_Spacelift employees should request reviews from the relevant engineers and are allowed to merge pull requests after they got at least one approval._

Thank you for your contribution! 🙇

### CODE REVIEW

File: docs/concepts/blueprint/README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ Published means that the blueprint is ready to be used. You can publish a bluepr
 
 A published blueprint cannot be moved back to draft state. You need to clone the blueprint, edit it and publish it.
 
+
+## Permissions
+
+Blueprints permissions are managed by [Spaces](../spaces/README.md). You can only create, update and delete a blueprint in a Space you have **admin** access to but they can be read by anyone with **read** access to the Space.
```
Comment (tomasmik) [inline docs/concepts/blueprint/README.md:39]: > You can only create, update and delete a blueprint in a Space you have **admin** access to but they can be read by anyone with **read** access to the Space.

`You can only create, update and delete a blueprint in a Space you have **admin** access to, but a blueprint can be read by anyone with **read** access to the Space.`

O something like that, I think we're mixing plural and singular there a bit with the `they`

File: docs/concepts/blueprint/README.md [hunk 2]
```diff
@@ -33,6 +33,13 @@ Published means that the blueprint is ready to be used. You can publish a bluepr
...
+
+Once the blueprint is published and you want to create a stack from it, the **read** access will be enough as long as you have **admin** to the Space where the stack will be created.
```
Comment (tomasmik) [inline docs/concepts/blueprint/README.md:41]: > enough as long as you have **admin** to the Space 

`as long as you are an admin` or `as long as you have admin rights in the space` or something? 🤷🏻  `you have **admin** to the Space` is not really correct I think.

**Review by tomasmik — APPROVED** [review]

File: docs/concepts/blueprint/README.md (hunk 1 above)
Comment (peterdeme) [reply docs/concepts/blueprint/README.md]: fixed

File: docs/concepts/blueprint/README.md (hunk 2 above)
Comment (peterdeme) [reply docs/concepts/blueprint/README.md]: ah I didn't mean it, probably Copilot haha, fixed

**Review by tomasmik — APPROVED** [review]

**Pull request merged**